src/network/networkd-dhcp6.c

```diff
@@ -300,8 +302,19 @@ static int dhcp6_lease_pd_prefix_acquired(sd_dhcp6_client *client, Link *link) {
                                 continue;
                         }
 
-                        route_add(link, AF_INET6, &pd_prefix, pd_prefix_len,
-                                  0, 0, 0, &route);
+                        table = link->network->dhcp_route_table;
+                        if (!link->network->dhcp_route_table_set && link->network->vrf)
+                                table = VRF(link->network->vrf)->table;
+
+                        r = route_add(link, AF_INET6, &pd_prefix, pd_prefix_len,
+                                      0, 0, table, &route);
+                        if (r < 0) {
+                                log_link_warning_errno(link, r, "Failed to add unreachable route for DHCPv6 delegated subnet %s/%u: %m",
+                                                       strnull(buf),
```

Comment:
Should not be the log like "Failed to add route "then errno should explain why failed ?

Reply:
Hmm? What do you mean? The message already contains `%m`.

Reply:
```unreachable ``` should not be there.

Reply:
Why? The route will be set to `RTN_UNREACHABLE`. Of course, at this time the type is not set...
Note that I just copied the message from the above.

Reply:
yes .